src/platforms/rcore_desktop_glfw.c

```diff
@@ -1873,6 +1873,13 @@ static void MouseCursorPosCallback(GLFWwindow *window, double x, double y)
     CORE.Input.Mouse.currentPosition.y = (float)y;
     CORE.Input.Touch.position[0] = CORE.Input.Mouse.currentPosition;
 
+    static bool firstMouseInput = true;
```

Comment:
This is one of the most horrible things I've seen in a long time. I'm afraid I'm not adding this dirty hack to raylib.

Reply:
Just for clarification, does it happen with other platforms?

Reply:
As said in the other thread, this issue is not limited to macOS. I'm seeing the same issue on Linux.

Reply:
Thanks for clarification, still thinking it should be addressed at GLFW level.